sort.html

```diff
@@ -12,26 +14,18 @@
           
             <div id="header-inner">
                 <a href="#" id="logo"></a>
-                <a class="navbar-brand" rel="home" href="#" title="logo">
-                    <img src="src/assets/logo.png" width="90" height="80" >
+                <a class="navbar-brand" rel="home" href="#" title="Strucutrex">
+                    <img src="src/assets/nlogo.png" class="main-logo" >
+                    <href="#" class="name">Structurex</a>
                 </a>
-                <div class="container blue circleBehind">
-                    <a href="#" id="menu-icon"></a>
-                    <a href="index.html" class="current">Home</a>
-                    <a href="#">Sorting</a>
-                    <a href="search.html">Searching</a>
-                    <a href="#">Path Finding</a>
-                    <a href="#">Graph</a>
-					<div class="dropdown">
-						<button onclick="show_hide()" class="Morebutton" > More </button>
```

Comment:
Here.

Reply:
@akshitadixit  Done ma'am . Made all the requested changes.  🙂